test/verify/check-metrics

```diff
@@ -442,6 +444,67 @@ class TestCurrentMetrics(MachineCase):
         # this settles down slowly, don't wait for becoming really quiet
         with b.wait_timeout(180):
             b.wait(lambda: float(b.text("#load-avg").split()[2].rstrip(',')) < 10)
+            b.wait_in_text(".metrics-history", "spike")
+
+        # Check logs for given CPU spike
+        (hour, minute) = self.browser.call_js_func("""(function () {
+            let minute = -1;
+            let hour = -1;
+            const hours = document.getElementsByClassName("metrics-hour");
+            Array.from(hours).some(h => {
+                const minutes = h.getElementsByClassName("metrics-minute");
+                return Array.from(minutes).some(m => {
+                    if (Array.from(m.getElementsByClassName("type")).length) {
+                        minute = m.getAttribute("data-minute");
+                        hour = h.id.split("-")[2];
+                        return true;
+                    }
+                    return false;
+                });
+            });
+
+            return [hour, minute];
+        })""")
+
+        hour = int(hour)
+        minute = int(minute)
+
+        b.click("#metrics-hour-{0} div.metrics-minute[data-minute={1}] button".format(hour, minute))
+        b.wait_visible(".cockpit-log-panel")
+        b.wait_visible(".description button:contains('All logs')")
+        logs = self.browser.call_js_func("ph_count", ".cockpit-logline")
+        if (logs == 0):
+            # In case there are no log messages lets check with journal it is correct
+            hour = hour / 1000 + minute * 60
+            lines = m.execute("journalctl --since @{0} --until @{1} --quiet".format(hour, hour + 60)).strip().split("\n")
+            self.assertEqual(len(lines), 0)
+        else:
+            # There are some logs, lets check they are valid
+            b.click(".cockpit-logline:first-child")
+            b.enter_page("/system/logs")
+            b.wait_visible(".info-table-ct tr td:contains('__REALTIME_TIMESTAMP')")
+
+            # Log timestamp needs to be in reasonable range
+            timestamp = int(b.text(".info-table-ct tr td:contains('__REALTIME_TIMESTAMP') + td")) / 1000
+            self.assertGreaterEqual(timestamp, hour + (minute - 1) * 60000) # -1 as if the event happens at X:00 we check X-1 minute as well
+            self.assertLessEqual(timestamp, hour + (minute + 1) * 60000)
+
+            # Check number of logs and remember location
+            b.click("a:contains('Logs')")
+
+            if (logs >= 10):
```

Comment:
Same here -- tests should be assertions, not conditions. The test could submit some logs, or do something like 

    systemd-run --unit timeout 30 eatmycpu dd if=/dev/urandom of=/dev/null

then query the unit's start time with `sytemctl show`, and then ensure that you eventually see the systemd message in the journal in the expected hour.